<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.dell.delta_uat_automation.testcases.TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX" tests="1" time="594.864" ignored="0" failures="1" timestamp="27 Feb 2017 06:46:19 GMT" skipped="0" hostname="W10DMLDT12" errors="0">
  <testcase name="validate_TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX" time="594.864" classname="com.dell.delta_uat_automation.testcases.TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX">
    <failure type="java.lang.AssertionError" message="caught exception while executing SelectServiceType_ServiceOptionUx method">
      <![CDATA[java.lang.AssertionError: caught exception while executing SelectServiceType_ServiceOptionUx method
	at org.testng.Assert.fail(Assert.java:83)
	at com.dell.delta_uat_automation.workflows.NewDispatchWorkFlow.SelectServiceType_ServiceOption_UpgradeSLAUx(NewDispatchWorkFlow.java:2035)
	at com.dell.delta_uat_automation.workflows.NewDispatchWorkFlow.create_Dispatch_Upgrade_SLA_LA(NewDispatchWorkFlow.java:2056)
	at com.dell.delta_uat_automation.testcases.TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX.validate_TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX(TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.openqa.selenium.NoSuchWindowException: No window found (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 1.43 seconds
Build info: version: 'unknown', revision: '31c43c8', time: '2016-08-02 21:57:56 -0700'
System info: host: 'W10DMLDT12', ip: '10.116.12.5', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=false, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, version=11, platform=WINDOWS, nativeEvents=false, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:25021/, javascriptEnabled=true, ignoreProtectedModeSettings=true, enableElementCacheCleanup=true, unexpectedAlertBehaviour=accept}]
Session ID: 1a9ee63d-ee47-4219-a14d-ef8e1d2e46ae
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:683)
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.window(RemoteWebDriver.java:1020)
	at com.dell.delta_uat_automation.workflows.NewDispatchWorkFlow.SelectServiceType_ServiceOption_UpgradeSLAUx(NewDispatchWorkFlow.java:2014)
	... 27 more
]]>
    </failure>
  </testcase> <!-- validate_TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX -->
</testsuite> <!-- com.dell.delta_uat_automation.testcases.TC_1341385_LA_Confirm_Service_Details_Upgrade_SLA_UX -->
